connector/util/client/configuration.go

```diff
@@ -39,12 +52,36 @@ func init() {
 		QueryTimeoutSeconds = 3 * 60
 	}
 
+	QueryToolsTimeoutSeconds, err = strconv.ParseInt(os.Getenv("QUERY_TOOLS_TIMEOUT_SECONDS"), 10, 64)
```

Comment:
The following new environment variables:
- QUERY_TOOLS_TIMEOUT_SECONDS
- SURVIVAL_ANALYSIS_TIMEOUT_SECONDS
- TOKEN_TIMEOUT_SECONDS
- WAIT_TICK_SECONDS

need to be added in the following files:
- build/package/medco/Dockerfile (sorted in the proper section according to where they are used, just like all the others), with the default values set
- deployments/docker-compose-definitions.yml (in the container where they are used, in sync with where they were put in the common Dockerfile), with the default values set again